collectors/rest/duo/hmac_conf.json

```diff
@@ -0,0 +1,13 @@
+{
+  "stringBuilders": [
+    "headers['x-duo-date']",
+    "method.toUpperCase()",
+    "urlObj.hostname",
+    "urlObj.pathname",
+    "urlObj.search.substring(1).replace(/\\+/g, '%20') || '' /* convert spaces that were endcoded as '+' to the url encoded space value */"
+  ],
+  "headerName": "authorization",
+  "headerExpression": "'Basic ' + C.Encode.base64(`DIX6AAR2DWSXSCFDDY77:${C.Crypto.createHmac(signatureString, '1tpkYv8hlzdz7fuJZXumhxy77Y8pyU9IpcTDOd9X', 'sha1', 'hex')}`)",
```

Comment:
ah whoops...forgot to fix this :face_exhaling: 
yay for accidentally leaking secrets. Good thing I can reset the key